parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         - "--tls-private-key-file=/etc/kubernetes/certs/apiserver.key"
         - "--client-ca-file=/etc/kubernetes/certs/ca.crt"
         - "--service-account-key-file=/etc/kubernetes/certs/apiserver.key"
+        - "--oidc-client-id=<oidcClientId>"
+        - "--oidc-issuer-url=<oidcIssuerUrl>"
+        - "--oidc-username-claim=oid"
```

Comment:
I think we'd prefer that these options be entirely omitted at runtime if not enabled. This is a more explicit approach, and also ensures that if a future bug is introduced that breaks variable injection we are more able to easily identify `--oidc-client-id=` (empty value) as a symptom of something wrong.

Here is an example in a recent PR of an option line being removed entirely during template generation if a given condition isn't met:

https://github.com/Azure/acs-engine/commit/f5c988a5031de8ce39009351d6801a0bd8b95eeb#diff-b7e839779245b296d836bd5663a27941R237

I hope that makes sense!